examples/v2beta1/dra/README.md

```diff
@@ -0,0 +1,5 @@
+# DRA enabled Kubernetes
```

Comment:
I think this should be a separate PR once the upgrade goes in.

DRA support on OSS projects is quite difficult at the moment. You need to guarantee that someone has the feature gates on (which is not yet possible in any k8s cluster by default), nvidia dra driver is installed and hope that the APIs for this don't change over time.

Also, these examples would require nvidia gpus of course. I tend to prefer examples on OSS projects to be CPU based in cases where performance doesn't matter. I don't know how this project keeps examples up to date and working. In other areas, we are exploring the idea of using the sample driver for DRA to at least show how DRA works without needing GPU access to actually run this.

Reply:
+1
Additionally, I do not want to block the new release by this PR.
So, could you focus on lib bumping in the PR?

Reply:
Dropped